test/test_mps.py

```diff
@@ -1548,6 +1548,12 @@ def test_storage_offset_greater_than_src_nbytes(self):
     def test_bool_full(self):
         x = torch.full((3, 3), True, device='mps')
 
+    # Empty unary op should return tensor of the same size
```

Comment:
I think testing TestConsistency() would cover empty tensors.

Reply:
Not sure I understand the comment, `neg` is already tested, but looks like not over empty tensors, see
https://github.com/pytorch/pytorch/blob/9e81c0c3f46a36333e82b799b4afa79b44b6bb59/test/test_mps.py#L6279-L6282